calculation_rate_hz: 200

odom_input_topic: "/t265/odom/sample"

odom_output_topic: "/odom"
odom_output_frame: "odom"

base_frame: "base_footprint"

sensor_offset:
  position: 
    x: 0.0
    y: 0.0
    z: 0.0

  rotation: 
    x: 0.0
    y: 0.0
    z: 0.0

ignore_x: false
ignore_y: false
ignore_z: true